txhttptrace.py

```diff
@@ -21,10 +21,12 @@ def my_handler(ctx, request, ...):
 _logger = None
 _debug = False
 _exceptions_to_ignore = []
+_log_errors_as_warnings = False
 
-def set_logger(logger):
+def set_logger(logger, log_errors_as_warnings=False):
     global _logger
     _logger = logger
+    _log_errors_as_warnings = log_errors_as_warnings
```

Comment:
Why allow this to be configurable? Is there any use case? Otherwise it's better to just decide to do it this way, then people won't need to decide which to use.